tech.json

```diff
@@ -53,6 +53,32 @@
               "note": "Using the momentum from jumping through a doorway from normal to water physics."
             }
           ]
+        },
+        {
+          "name": "canPlayInSand",
+          "requires": [ "h_canNavigateUnderwater" ],
```

Comment:
tbh leaving the suitless requirement here was so that I didn't break anything while working on other rooms.

Non water physics sand would behave as if you have gravity and does feel unrelated to a suitlessMaridia requirement.  `canEscapeSand` is a problem because of the water physics so suitlessMaridia can stay there.